src/main/java/com/blamejared/jeitweaker/zen/recipe/JeiRecipe.java

```diff
@@ -0,0 +1,67 @@
+package com.blamejared.jeitweaker.zen.recipe;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.jeitweaker.zen.category.JeiCategory;
+import com.blamejared.jeitweaker.zen.component.RawJeiIngredient;
+import org.openzen.zencode.java.ZenCodeType;
+
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+// TODO("Expose methods?")
+@Document("mods/JEI/Recipe/JeiRecipe")
+@ZenCodeType.Name("mods.jei.recipe.JeiRecipe")
+@ZenRegister
+public final class JeiRecipe {
```

Comment:
With further consideration, there might actually be no reason to expose any of these methods. I'm going to leave the class exposed just in case a JeiTweaker addon wants to give people access to custom JeiRecipes, just in case.